### PR TITLE
Add 1.0.x migration guide

### PR DESCRIPTION
**Pull request by sq2gxo**

### CODE REVIEW

**Review by karolkieglerski — APPROVED** [review]

File: MIGRATION-V1.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Migration from 0.1.x to 1.0.x
```
Comment (mfranczak) [inline MIGRATION-V1.md:1]: Please include information that token needs to be updated due to new permissions.

File: MIGRATION-V1.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+```
+Remove old configuration file (actovation-config.yaml)
+Install new `dynatrace-gcp-function` deployment following instructions in [Dynatrace documentation](https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-cloud-platforms/google-cloud-platform/set-up-integration-gcp/deploy-as-gcp-function/) 
+
+## Dynatrace environment cleanup
+There might be some Dynatrace dashboards and/or alerts left from previous 0.1.x installations. They need to be deleted manually.
+New installation o 1.0.x version will add new dashboards and alerts.
+
+## List of incompatible changes
+Following GCP metrics dimension names has changed in 1.0.x. If you created own dashboards or alerts based on GCP metrics you need to update them.
```
Comment (mfranczak) [inline MIGRATION-V1.md:23]: Please add visual indicators for If you created own dashboards or alerts based on GCP metrics you need to update them. 

⚠️ If you created own dashboards or alerts based on GCP metrics you need to update them

**Review by mfranczak — CHANGES_REQUESTED** [review]

File: MIGRATION-V1.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+| topic_id | gcp.instance.id |
+| key_id | gcp.instance.id |
+| worker_id | gcp.instance.id |
+| agent_id | gcp.instance.id |
+| gateway_id | gcp.instance.id |
+| name | gcp.instance.name |
+| autoscaler_name | gcp.instance.name |
+| environment_name | gcp.instance.name |
+| cluster_name gcp.instance.name | gcp.instance.name |
+| function_name gcp.instance.name | gcp.instance.name |
```
Comment (mfranczak) [inline MIGRATION-V1.md:55]: are we replacing function_name with gcp.instance.name? Or we keep function_name introduce gcp.instance.name?

Comment (sq2gxo) [reply MIGRATION-V1.md:55]: Yes, we are replacing old dimension 'function_name' with new one 'gcp.instance.name'

**Review by Dawid-Kaszubski — APPROVED** [review]

**Pull request merged**